docs/pages/guides/mesheryctl/system-commands.md

```diff
@@ -0,0 +1,183 @@
+---
+layout : default
+title : Mesheryctl system commands
+permalink : guides/mesheryctl/system-commands
+language : en
+type : Guides
+category : mesheryctl
+---
+
+Let's get familiar with mesheryctl system commands. The syntax of the mesheryctl commands goes as follws : `mesheryctl <Main_command> <Argument> <Flags>`
+
+## Main_command : system
+### start 
+`mesheryctl system start` : This will initiate Meshery & automatically open it in your default web browser.
+
+<a href="{{ site.baseurl }}/assets/img/syscmd/start.png"><img alt="skip-browser" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/syscmd/start.png" /></a>
+
+`mesheryctl system start --skip-browser` : It skips opening Meshery in your browser with the provided URL.
+
+<a href="{{ site.baseurl }}/assets/img/syscmd/skipbrowser.png"><img alt="skip-browser" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/syscmd/skipbrowser.png" /></a>
+
+`mesheryctl system start --skip-update` : It is used when you want to skip updating Meshery if an update is available.
+
+<a href="{{ site.baseurl }}/assets/img/syscmd/system update.png"><img alt="skip-browser" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/syscmd/system update.png" /></a>
+
+`mesheryctl system start --reset` : It resets your Meshery configuration file to its default configuration.
+
+`mesheryctl system start --platform string` : It allows you specify a platform for deploying Meshery.
+
+<a href="{{ site.baseurl }}/assets/img/syscmd/platform.png"><img alt="skip-browser" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/syscmd/platform.png" /></a>
+
+
+### stop 
+`mesheryctl system stop` : It stops Meshery resources & delete its associated namespaces.
+
+<a href="{{ site.baseurl }}/assets/img/syscmd/stop.png"><img alt="skip-browser" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/syscmd/stop.png" /></a>
+
+`mesheryctl system stop --reset` : It stops Meshery and resets the Meshery configuration file to its default configuration.
+
+<a href="{{ site.baseurl }}/assets/img/syscmd/stop reset.png"><img alt="skip-browser" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/syscmd/stop reset.png" /></a>
+
+`mesheryctl system stop --keep-namespace` : It stops Meshery without deleting the associated namespaces.
+
+<a href="{{ site.baseurl }}/assets/img/syscmd/keep namespace.png"><img alt="skip-browser" style="width:500px;height:auto;" src="{{ site.baseurl }}/assets/img/syscmd/keep namespace.png" /></a>
+
+`mesheryctl system stop --force` : Force stops Meshery instead of gentle way. This is only in emergency situations when `mesheryctl system stop` can't halt Meshery.
```

Comment:
```suggestion
`mesheryctl system stop --force` : Force stops Meshery instead of gentle way. This is only used in emergency situations when `mesheryctl system stop` can't halt Meshery.
```